docs/tools/testing/testing_greentea.md

```diff
@@ -178,6 +314,191 @@ This detects your attached target and drives the test. If you need to rerun the
 
 For an explanation of the arguments used in this command, please run `mbedhtrun --help`.
 
+### Command-line usage
+
+This section highlights a few of the capabilities of the Greentea command-line interface. For a full list of the available options, please run `mbed test --help`.
+
+#### Listing all tests
+
+You can use the `-l` argument to list all available tests:
+
+```
+$ mbed test -l
+[mbed] Working path "/Users/janjon01/repos/first-greentea-test" (program)
+Test Case:
+    Name: mbed-os-components-storage-blockdevice-component_flashiap-tests-filesystem-fopen
+    Path: ./mbed-os/components/storage/blockdevice/COMPONENT_FLASHIAP/TESTS/filesystem/fopen
+Test Case:
+    Name: mbed-os-features-cellular-tests-api-cellular_device
+    Path: ./mbed-os/features/cellular/TESTS/api/cellular_device
+
+...
+```
+
+#### Executing all tests
+
+The default action of Greentea using `mbed test` is to execute all tests that are found. You can also add `-v` to make the output more verbose.
+
+#### Limiting tests
+
+You can select test cases by name using the `-n` argument. This command executes all tests named `tests-mbedmicro-rtos-mbed-mail` from all builds in the test specification:
+
+```
+$ mbed test -n tests-mbedmicro-rtos-mbed-mail
+```
+
+When using the `-n` argument, you can use the `*` character at the end of a test name to match all tests that share a prefix. This command executes all tests that start with `tests-mbedmicro-rtos`:
```

Comment:
This will now use "globbing" rules, so you can actually sprinkle the `*` characters any where you like.